data/categories.txt

```diff
@@ -6485,6 +6485,51 @@ sw:Malazi
 fa:پناهگاه
 mr:निवारा
 
+amenity-shelter-basic_hut|@category_hotel
+en:3Bivouac
+ar:إقامة مؤقتة
+be:бівуак
+bg:бивак
+ca:vivac
+cs:bivak
+da:Bivuak
+de:4Biwakschachtel|Biwak
+el:στρατοπεδεύω
+en-US:bivouac
+es:vivac
+et:bivaak
+eu:bibaka
+fa:بیواک
+fi:ulkomajoitus
+fr:bivouac
+he:מַאֲהָל
+hi:पड़ाव
+hu:szabadban alszik
+id:bivak
+it:bivacco
+ja:ビバーク
+ko:비박
+mr:बिव्होक
+nb:bivuakk
+nl:bivak
+pl:biwak
+pt:bivaque
+ro:bivuac
+ru:Бивуак
+sk:bivak
+sv:Bivack
+sw:bivouac
+th:พักแรม
+tr:çadır
+uk:бівуак
+vi:bivouac
+zh-Hans:行军床
+zh-Hant:露營
+
+amenity-shelter-lean_to|@category_hotel
+en:3Lean-to Shelter
+de:Lean-to Wetterschutz
```

Comment:
I've really got no idea if there's a term for this, but I can just describe it literally as a leaning shelter: 

```
he = מחסה נשען
```

Reply:
Its not :)
Check the comments in types_strings.txt, e.g.
`comment = A traditional 3-walled shelter (one side open), suitable for overnight camping.`

Also see https://github.com/organicmaps/organicmaps/pull/7785#issuecomment-2050231094